src/RatingForm/RatingForm.js

```diff
@@ -1,20 +1,33 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Route, Switch, Link, NavLink } from 'react-router-dom';
 import './RatingForm.css';
+import { userRatingPost } from '../apiCalls.js';
 
 class RatingForm extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
       userRating: ''
     }
   }
 
-  updateValue(event) {
+  updateValue = (event) => {
     this.setState({userRating: event.target.value});
   }
 
+  addUserRating = (event) => {
+    event.preventDefault();
+    const userID = this.props.userInfo.id;
+    const rating = {
+      movie_id: this.props.movieInfo.id,
+      rating: parseInt(this.state.userRating)
+    };
+    userRatingPost(userID, rating)
+    .then(data => console.log(data))
+    .catch(error => this.setState({error: error.message}))
```

Comment:
We need to refactor other catch methods to add the error message to state like this.